lib/MathChangeTypes.js

```diff
@@ -1,76 +1,75 @@
 // The text to identify rules for each possible step that can be taken
 
 module.exports = {
-  NO_CHANGE: 'No change',
+  NO_CHANGE: 'NO_CHANGE',
```

Comment:
can you add the examples you use in Athena here? would make it very helpful for people to know what they do